src/pages/Send.tsx

```diff
@@ -98,7 +89,10 @@ export default function Send() {
         (address) => commands.validateAddress(address).catch(addError),
         'Invalid address',
       ),
-    amount: positiveAmount(asset?.decimals || 12),
+    amount: positiveAmount(asset?.decimals || 12).refine(
+      (amount) => asset ? BigNumber(amount).lte(toDecimal(asset.balance, asset.decimals)) : true,
```

Comment:
This doesn't take into account the fee

Reply:
Correct, I figured since the send covers both cat and xch that fees checking should be done on the fees themselves against the xch balance but that was out of scope for what I was adding here (just making sure the amount is less than current balance of that token)